controllers/errors.controller.js

```diff
@@ -4,6 +4,14 @@ exports.handle400s = (err, _, res, next) => {
   } else next(err);
 };
 
+exports.handle404s = (err, _, res, next) => {
+    if (err.code === '22003') {
+        console.log("in the handle 404s")
```

Comment:
try to remove console.logs before commits.

Reply:
Hi Nick, was it all good apart from including that console log? Or are you still looking at it?

Cheers